test/test_transform_collections_mixed.cpp

```diff
@@ -1455,3 +1461,261 @@ TEST(Transform_ClearDictionaryInsideDictionaryVsCreateDictionary)
         CHECK_EQUAL(dict->get("key2"), 2);
     });
 }
+
+TEST(Transform_UpdateClearVsUpdateClear)
+{
+    std::vector<int> timestamps{1, 2, 3, 4};
+
+    do {
+        // Baseline: property 'any' is not set to any type
+        // {id: 1, any: null}
+        TransformTestHarness h(test_context, {}, [](Obj, ColKey) {});
```

Comment:
nit, can we initialize the harness after we check if the timestamps are valid so we don't waste time spinning up a server/clients for cases we aren't going to test?